From e622fed1f32a0ab6db08419cc61dea4d8256eec6 Mon Sep 17 00:00:00 2001
From: Tejun Heo <tj@kernel.org>
Date: Fri, 23 Feb 2024 09:57:57 -1000
Subject: [PATCH 142/152] scx: Reorder the two fields in scx_init_task_args

So that more important and always existing .fork is the first one.
---
 include/linux/sched/ext.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/include/linux/sched/ext.h b/include/linux/sched/ext.h
index 474131a20..d3ccf4622 100644
--- a/include/linux/sched/ext.h
+++ b/include/linux/sched/ext.h
@@ -127,15 +127,15 @@ enum scx_ops_flags {
 
 /* argument container for ops.init_task() */
 struct scx_init_task_args {
-#ifdef CONFIG_EXT_GROUP_SCHED
-	/* the cgroup the task is joining */
-	struct cgroup		*cgroup;
-#endif
 	/*
 	 * Set if ops.init_task() is being invoked on the fork path, as opposed
 	 * to the scheduler transition path.
 	 */
 	bool			fork;
+#ifdef CONFIG_EXT_GROUP_SCHED
+	/* the cgroup the task is joining */
+	struct cgroup		*cgroup;
+#endif
 };
 
 /* argument container for ops.exit_task() */
-- 
2.43.0.232.ge79552d197

